### PR TITLE
*/*: add EAPI guards

### PR DESCRIPTION
**Pull request by thesamesam**

### CODE REVIEW

**Review by SoapGentoo — APPROVED** [review]

File: eclass/apache-module.eclass
Comment (SoapGentoo) [inline eclass/apache-module.eclass:123]: remove

File: eclass/cron.eclass
Comment (SoapGentoo) [inline eclass/cron.eclass:136]: remove?

File: eclass/pam.eclass
Comment (SoapGentoo) [inline eclass/pam.eclass:96]: remove

File: eclass/cron.eclass
Comment (ulm) [reply eclass/cron.eclass:136]: Sure, that change by itself is fine, but many such changes increase the risk that there will be some breakage.

I'd rather do only the minimum that is necessary. The rest is not urgent and can be done later (preferably by eclass maintainers if they are still active).

File: eclass/java-osgi.eclass
Comment (ulm) [inline eclass/java-osgi.eclass:32]: Chances are that this will never happen (we did the opposite).
So, either leave that comment alone, or remove it altogether.

File: eclass/multilib.eclass
Comment (ulm) [inline eclass/multilib.eclass:16]: There's another EAPI test in line 56 which could be simplified.

File: eclass/openib.eclass
Comment (ulm) [inline eclass/openib.eclass:13]: versionator doesn't support EAPI 7.

File: eclass/java-utils-2.eclass
Comment (ulm) [inline eclass/java-utils-2.eclass:30]: versionator doesn't support EAPI 7.

File: eclass/xemacs-packages.eclass
Comment (ulm) [inline eclass/xemacs-packages.eclass:67]:
```suggestion
	cd "${ED}${install_dir}" || die
```
`install_dir` is assigned just above, and is guaranteed to start with a `/`.

File: eclass/openib.eclass
Comment (ulm) [inline eclass/openib.eclass:14]: I think you cannot support EAPI 7 without additional changes.

For example, there is this line in global scope which will need versionator.eclass:
```
OFED_BASE_VER=$(get_version_component_range 1-3 ${OFED_VER})
```
(There may be more, I haven't checked.)

File: eclass/eutils.eclass
Comment (ulm) [inline eclass/eutils.eclass:94]: More such case statements below, where EAPIs 0 to 4 could be removed.

File: eclass/openib.eclass
Comment (thesamesam) [reply eclass/openib.eclass]: You're right.

File: eclass/gnome.org.eclass
Comment (ulm) [inline eclass/gnome.org.eclass:31]: "defaults to defaults to"

While at it, rewrap the long line to stay below the 80 char limit.

File: eclass/stardict.eclass
Comment (ulm) [inline eclass/stardict.eclass:8]:
```suggestion
# @BLURB: Convienence eclass to do stardict dictionary installations
```

`@BLURB` must be in one line.

File: eclass/stardict.eclass
Comment (ulm) [inline eclass/stardict.eclass:15]: Pretty sure that eclassdoc will rewrap this into a single paragraph.

File: eclass/toolchain-funcs.eclass
Comment (mgorny) [inline eclass/toolchain-funcs.eclass:17]: Now, this is going to block mah python-r1 EAPI 8 support :-(.

Comment (ulm) [reply eclass/toolchain-funcs.eclass:17]: Should be fixed once PR #21355 is merged.

**Pull request closed without merging**
